odbc/src/api/functions.rs

```diff
@@ -1095,17 +1096,59 @@ pub unsafe extern "C" fn SQLFreeStmt(_statement_handle: HStmt, _option: SmallInt
 /// [`SQLGetConnectAttr`]: https://learn.microsoft.com/en-us/sql/odbc/reference/syntax/SQLGetConnectAttr-function
 ///
 /// # Safety
-/// Because this is a C-infereface, this is necessarily unsafe
+/// Because this is a C-interface, this is necessarily unsafe
 ///
 #[no_mangle]
 pub unsafe extern "C" fn SQLGetConnectAttr(
```

Comment:
Leave `SQLGetConnectAttr` unimplemented and move this code to `SQLGetConnectAttrW` since you are outputting *WChar